src/main/kotlin/org/cascadebot/cascadebot/commands/management/AutoRoleCommand.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.management
+
+import net.dv8tion.jda.api.entities.Member
+import net.dv8tion.jda.api.entities.Role
+import org.cascadebot.cascadebot.commandmeta.CommandContext
+import org.cascadebot.cascadebot.commandmeta.MainCommand
+import org.cascadebot.cascadebot.commandmeta.Module
+import org.cascadebot.cascadebot.commandmeta.SubCommand
+import org.cascadebot.cascadebot.messaging.MessageType
+import org.cascadebot.cascadebot.messaging.MessagingObjects
+import org.cascadebot.cascadebot.permissions.CascadePermission
+
+class AutoRoleCommand : MainCommand() {
```

Comment:
We might want to have a list sub command as is convention instead of making the main command the list